src/Algebra/Structures.agda

```diff
@@ -862,10 +861,15 @@ record IsRing (+ * : Op₂ A) (-_ : Op₁ A) (0# 1# : A) : Set (a ⊔ ℓ) where
     )
 
   zeroˡ : LeftZero 0# *
-  zeroˡ = proj₁ zero
+  zeroˡ = Consequences.assoc+distribʳ+idʳ+invʳ⇒zeˡ setoid
+    +-cong *-cong +-assoc (proj₂ distrib) +-identityʳ -‿inverseʳ
 
   zeroʳ : RightZero 0# *
-  zeroʳ = proj₂ zero
+  zeroʳ = Consequences.assoc+distribˡ+idʳ+invʳ⇒zeʳ setoid
+    +-cong *-cong +-assoc (proj₁ distrib) +-identityʳ -‿inverseʳ
```

Comment:
Two comments on this, the substantive part of the issue/PR:
* it would good to merge #2206 and use the corrected/renamed versions of these lemmas from `Consequences` (we might debate the order of merging, but each PR along the lines of the present one adds to the knock-on viscosity associated with the renaming...)
* have you checked the 'most general' point in the `Structures`/`Bundles` hierarchy that this lemma can be placed? AFAICT, it can already be proved at `IsNearring`, but possibly more general still?

Reply:
Good point. I've moved the proof to `IsRingWithoutOne`. And yes the other PR will be merged as well.